### PR TITLE
add Air target to OpenFL/Lime projects

### PR DESCRIPTION
**Pull request by peteshand**

### CODE REVIEW

**Comment by SlavaRa** [comment]
@peteshand It isn't works as is, but don't worry :)
@jgranick Hi, Can you update https://gist.github.com/1763850 for AIR target or can you give new link to a full description of the project file?

**Comment by peteshand** [comment]
What error are you getting?

**Comment by SlavaRa** [comment]
```
Building OpenFLTestProject
Running Pre-Build Command Line...
cmd: "C:\HaxeToolkit_3.4.7\haxe/haxelib" run lime build "project.xml" air -release -Dfdb
[31;1mError:[0m [1mYou must define AIR_SDK with the path to your AIR SDK[0m
Build halted with errors.
Done(1)
```

**Comment by peteshand** [comment]
Ar yes, good point. I've added the following line to the openfl project.xml template.
<define name="AIR_SDK" value="${AIR_SDK}"></define>
This will still require the end user to have AIR_SDK set in their system path

File: FlashDevelop/Bin/Debug/Projects/380 Haxe - OpenFL Project/project.xml.template [hunk 1]
```diff
@@ -23,4 +23,6 @@
 	<!-- optimize output
 	<haxeflag name="-dce full" /> -->
 	
+	<define name="AIR_SDK" value="${AIR_SDK}"></define>
```
Comment (SlavaRa) [inline FlashDevelop/Bin/Debug/Projects/380 Haxe - OpenFL Project/project.xml.template:26]:
```xml
<!-- Support for Adobe AIR: http://community.openfl.org/t/official-support-for-adobe-air/9468 -->
<!--<define name="AIR_SDK" value="path/to/your/air/sdk" if="air" /> -->
```
I think that true way is edit `~/.lime/config.xml` and add the following:
`<set name="AIR_SDK" value="path/to/your/air/sdk" />`

**Pull request closed without merging**